app/controllers/v2/repositories_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module V2
+  # Handles all requests for repository CRUD operations
+  class RepositoriesController < ResourcesController
+    find_param :slug
+    permitted_params :name, :description
```

Comment:
I believe this also allows the name to be edited. Is that wanted? If not, we need to forbid that in the `update` action.